src/_articles/archive/zones.md

```diff
@@ -21,58 +21,49 @@ obsolete: true
 
 ### Asynchronous dynamic extents
 
-_Written by Florian Loitsch and Kathy Walrath<br>
-March 2014_
-
-This article discusses zone-related APIs in the dart:async library,
-with a focus on the top-level `runZoned()` function.
+This article discusses zone-related APIs in the [dart:async][] library,
+with a focus on the top-level [`runZoned()`][]
+and [`runZonedGuarded()`][] functions.
 Before reading this article,
 you should be familiar with the techniques covered in
 [Futures and Error Handling](/guides/libraries/futures-error-handling).
 
-Currently, the most common use of zones is
-to handle errors raised in asynchronously executed code.
-For example,
-a simple HTTP server
-might use the following code:
-
-{% prettify dart tag=pre+code %}
-[!runZoned(() {!]
-  HttpServer.bind('0.0.0.0', port).then((server) {
-    server.listen(staticFiles.serveRequest);
-  });
-[!},
-onError: (e, stackTrace) => print('Oh noes! $e $stackTrace'));!]
-{% endprettify %}
-
-Running the HTTP server in a zone
-enables the app to continue running despite uncaught (but non-fatal)
-errors in the server's asynchronous code.
-
-{{site.alert.info}}
-  **API note:**
-  This use case won't always require zones.
-  We expect that isolates will have an API enabling you to
-  listen for uncaught errors.
-{{site.alert.end}}
+[dart:async]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/dart-async-library.html)
+[`runZoned()`]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/runZoned.html)
+[`runZonedGuarded()`]: ({{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/runZonedGuarded.html)
 
 Zones make the following tasks possible:
```

Comment:
Skipping because it reads like an instruction, this article is purely informational. And the tasks listed aren't only possible by using zones, in fact, zones aren't even recommended for some of them.